index-4.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project HTML CSS</title>
+    <link rel="stylesheet" href="style4.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Sedgwick+Ave+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="present">
+        <div class="menu">
+            <div>
+                <img src="chicken.png" alt="cool chicken with sunglasses" class="logo-chicken">
+            </div>
+            <div class="logo-name">
+                <h1>Urban Hen Hangout</h1>
+            </div>
+            <nav class="buttons">
+                <a href="index.html">Home</a>
+                <a href="index-2.html">About us</a>
+                <a href="index-4.html">Menu</a>
+                <a href="index3.html">Contact</a>
+            </nav>
+        </div>
+    </div>
+    <div class="meniu">
+        <h1 class="title">Our special offers </h1>
+        <br><br><br>
+        <div class="combo1">
+            <img src="chest.png" alt="bbq Chicken with rice" class="food1">
+            <div class="description1">
+                <h1 class="food-title-1">BBQ chicken with rice</h1>
+                <p class="description">Savor the mouthwatering delight of our tender BBQ Chicken, glazed to perfection and served alongside aromatic rice. A harmonious blend of smoky, savory flavors that will satisfy your cravings and leave you longing for more.</p>
+                <button class="price">7.99$</button>
+            </div>
+        </div>
+        <div class="combo2">
+            <img src="Crispy chicken.png" alt="Crispy chicken with salad" class="food2">
+            <div class="description2">
+                <h1 class="food-title-2">Crispy chicken with salad</h1>
+                <p class=>Indulge in the irresistible crunch of our Crispy Chicken, a golden masterpiece that pairs exquisitely with a fresh, vibrant salad. A delightful symphony of textures and flavors that will elevate your dining experience to new heights</p>
```

Comment:
Ai uitat să pui clasă aici, dacă era nevoie.

```suggestion
                <p>Indulge in the irresistible crunch of our Crispy Chicken, a golden masterpiece that pairs exquisitely with a fresh, vibrant salad. A delightful symphony of textures and flavors that will elevate your dining experience to new heights</p>
```